gpflow/covariances/kufs.py

```diff
@@ -1,16 +1,26 @@
 import tensorflow as tf
-from ..inducing_variables import InducingPoints, Multiscale, InducingPatches
+from ..base import TensorLike
+from ..inducing_variables import InducingVariables, InducingPoints, Multiscale, InducingPatches
 from ..kernels import Kernel, SquaredExponential, Convolutional
 from .dispatch import Kuf
 
 
-@Kuf.register(InducingPoints, Kernel, object)
-def _Kuf(inducing_variable: InducingPoints, kernel: Kernel, Xnew: tf.Tensor):
+@Kuf.register(InducingVariables, Kernel, TensorLike)
+def Kuf_fallback(inducing_variable, kernel, X):
+    from warnings import warn
+    warn('Kuf(inducing_variable, kernel, X) is deprecated, please use '
+         'Kuf(kernel, inducing_variable, X) instead.',
+         DeprecationWarning)
```

Comment:
Why did you add this warning? This isn't the case, is it? I.e. `Kuf(inducing_variable, kernel, X)` is *not* deprecated, right?